### PR TITLE
IDEN-298: add subjectPostition

### PR DESCRIPTION
**Pull request by ilya-korotya**

### CODE REVIEW

File: schemas/json-ld/iden3credential.json-ld [hunk 1]
```diff
@@ -39,7 +39,11 @@
         "credentialSubject": {
           "@id": "iden3-vocab:credentialSubject",
           "@type": "@id"
-        },              
+        },
+        "subjectPosition": {
+          "@id": "iden3-vocab:subjectPosition",
+          "@type": "xsd:string"
```
Comment (demonsh) [inline schemas/json-ld/iden3credential.json-ld:45]: Maybe it should be bolean?

Comment (ilya-korotya) [reply schemas/json-ld/iden3credential.json-ld:45]: I think for end user it will be confused.
`subjecPosition: true`. What is mean? xD

Comment (vladyslavmunin) [reply schemas/json-ld/iden3credential.json-ld:45]: no.subjectPosition : index is ok

Comment (ilya-korotya) [reply schemas/json-ld/iden3credential.json-ld:45]: done

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**